server/routes/upload.js

```diff
@@ -1,59 +1,74 @@
 const express = require("express");
 const passport = require("passport");
-const AWS = require('aws-sdk');
-const uuidv4 = require('uuid/v4');
+const AWS = require("aws-sdk");
+const uuidv4 = require("uuid/v4");
 const Contest = require("../models/Contest");
 const mongoose = require("mongoose");
 
+// const s3 = new AWS.S3({
+//   accessKeyId: process.env.ACCESS_KEY_ID,
+//   secretAccessKey: process.env.SECRET_ACCESS_KEY,
+//   signatureVersion: "v4",
+//   region: process.env.S3_REGION
+// });
+
 const s3 = new AWS.S3({
-  accessKeyId: process.env.ACCESS_KEY_ID,
-  secretAccessKey: process.env.SECRET_ACCESS_KEY,
-  signatureVersion: 'v4',
+  credentials: {
```

Comment:
i believe the accessKeyId and secretAccessKey properties in this configuration aren't used anymore (given the changes we did to .env files to get it to finally work). Can remove them and only have the S3 region now.